### DISCUSSION TITLE
Auto-delete shared note after a certain time period ...

### DISCUSSION THREAD

**Discussion by generic-user** [alangrainger/share-note#35] [Vote for upcoming features]
Set timeframe in frontend, after which the note automatically gets deleted from server.

Thanks, 🎉

**Comment by alangrainger** [comment]
Oooh I like it! I will implement this 💯

**Comment by alangrainger** [comment]
This is now live in 0.7.0:

https://github.com/alangrainger/share-note/releases/tag/0.7.0

**Reply by generic-user** [reply]
Wow, nice! U 🎸🎸🎸
